java/arcs/core/host/Particle.kt

```diff
@@ -16,6 +16,15 @@ import arcs.core.storage.api.Handle
 /** Base interface for all particles. */
 interface Particle {
 
+    /**
+     * This field contains a reference to all of the [Particle]'s handles that were declared in
+     * the manifest.
+     */
+    val handles: HandleHolder
+
+    /** Called the first time this [Particle] is instantiated in an [Arc]. */
```

Comment:
It would be good to mention examples of things that the particle author might find useful to perform during `onCreate`